composer/utils/dist.py

```diff
@@ -78,6 +78,16 @@ def get_local_rank() -> int:
     return local_rank
 
 
+def get_node_id() -> int:
+    """Returns the node ID for the current rank. For example, if there are 2 nodes, and 2 ranks per node, then
```

Comment:
Can you add a big caveat to this comment that this assumes an equal number of processes per node?